.github/workflows/merlin.yml

```diff
@@ -312,9 +319,17 @@ jobs:
       DOCKER_REGISTRY: ghcr.io
       DOCKER_IMAGE_TAG: "ghcr.io/${{ github.repository }}/merlin-pyfunc-base:${{ needs.create-version.outputs.version }}"
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v4
       - name: Build Pyfunc Server Base Docker
         run: docker build -t ${{ env.DOCKER_IMAGE_TAG }} -f python/pyfunc-server/docker/base.Dockerfile python
+      - name: Test Build Pyfunc Server Docker
```

Comment:
Same as above for batch predictor: https://github.com/caraml-dev/merlin/pull/547/files#r1529665602